### PR TITLE
Update DG and PPP

### PR DESCRIPTION
**Pull request by BenjoBurger**
Ensure format follows AB3, add class diagram for daily class

### CODE REVIEW

File: src/main/java/newsonthego/commands/DailyNewsCommand.java [hunk 1]
```diff
@@ -96,7 +96,7 @@ private static void saveDailyArticlesParser() {
             case "save":
                 save(dailyLine);
                 System.out.println("What do you want from me?\n" +
-                        INDENT + "To return to main, type in: back ");
+                        INDENT + "gTo return to main, type in: back ");
```
Comment (AhmedShaiyan) [inline src/main/java/newsonthego/commands/DailyNewsCommand.java:99]: Typo here!

**Review by AhmedShaiyan — CHANGES_REQUESTED** [review]
LGTM other than the typo!

File: src/main/java/newsonthego/commands/DailyNewsCommand.java (hunk 1 above)
Comment (BenjoBurger) [reply src/main/java/newsonthego/commands/DailyNewsCommand.java]: HAHAH thanks!

**Review by anneleong — APPROVED** [review]
lgtm

**Pull request merged**